sdk/compiler/daml-extension/src/extension.ts

```diff
@@ -79,7 +79,7 @@ export async function activate(context: vscode.ExtensionContext) {
       // Try to find a client for the virtual resource- if we can't, log to DevTools
       let foundAClient = false;
       for (let projectPath in damlLanguageClients) {
-        if (vrPath.startsWith(projectPath)) {
+        if (isPrefixOfVrPath(projectPath)) {
```

Comment:
Small fix missing from [this pr](https://github.com/digital-asset/daml/pull/20051/files)